changelog/26848.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
```

Comment:
Even though I removed the license call in 1.17, I thought a changelog might be prudent because we're changing behavior here.